### PR TITLE
D3ASIM-3548: Added validator for the standard deviation of the measur…

### PR DESCRIPTION
**Pull request by spyrostz**
…ement from the forecasted asset energy data.

### CODE REVIEW

File: d3a_interface/settings_validators.py [hunk 1]
```diff
@@ -79,6 +79,13 @@ def validate_global_settings(settings_dict):
         raise D3ASettingsException("Invalid value for grid_fee_type "
                                    f"({settings_dict['grid_fee_type']}).")
 
+    if ("relative_std_from_forecast_percent" in settings_dict and not
+            PercentageRangeLimit.min <=
+            settings_dict["relative_std_from_forecast_percent"] <=
+            PercentageRangeLimit.max):
+        raise D3ASettingsException(f'Invalid value for relative_std_from_forecast_percent '
+                                   f'({settings_dict["relative_std_from_forecast_percent"]}).')
```
Comment (hannesdiedrich) [inline d3a_interface/settings_validators.py:87]: Please use double quotes. Thanks!

**Review by hannesdiedrich — APPROVED** [review]
Minor comment, LGTM

File: d3a_interface/settings_validators.py (hunk 1 above)
Comment (spyrostz) [reply d3a_interface/settings_validators.py:87]: Will do so thanks!

**Review by fievelk — APPROVED** [review]

File: d3a_interface/settings_validators.py [hunk 2]
```diff
@@ -79,6 +79,13 @@ def validate_global_settings(settings_dict):
         raise D3ASettingsException("Invalid value for grid_fee_type "
                                    f"({settings_dict['grid_fee_type']}).")
 
+    if ("relative_std_from_forecast_percent" in settings_dict and not
+            PercentageRangeLimit.min <=
+            settings_dict["relative_std_from_forecast_percent"] <=
+            PercentageRangeLimit.max):
+        raise D3ASettingsException(f"Invalid value for relative_std_from_forecast_percent "
+                                   f"({settings_dict['relative_std_from_forecast_percent']}).")
```
Comment (fievelk) [inline d3a_interface/settings_validators.py:87]: Nit:
```suggestion
        raise D3ASettingsException("Invalid value for relative_std_from_forecast_percent "
                                   f"({settings_dict['relative_std_from_forecast_percent']}).")
```

**Review by hannesdiedrich — DISMISSED** [review]
LGTM

**Review by faizan2590 — DISMISSED** [review]
LGTM

**Pull request merged**